src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-size: 12px;
+  font-weight: 500;
+  width: 100%;
+  @include flex-center();
```

Comment:
The use of @include flex-center() might not be necessary for the entire header if it causes unwanted alignment issues. Ensure that this mixin is applied only where needed.